### DISCUSSION TITLE
Setting for the Editor Text Color?

### DISCUSSION THREAD

**Discussion by CGPGrey** [Akifyss/obsidian-border#312] [Ideas]
Feature request: option to change editor text color

Fantastic theme -- best one I've stumbled upon in years.   Only thing I can't find is an option to change the color of the text in the editor itself.   The tint color is _sort of_ that, but it's very faint.  Trying to achive that green-on-black terminal look.

**Comment by Akifyss** [comment]
You can set the Text Color in this section:
![image](https://github.com/user-attachments/assets/771ba1e1-f550-449b-a41b-c69fb51f5bdb)
